### DISCUSSION TITLE
limitiation data when serach column on laravel backpack

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#693] [Q&A (Help)]
```

    $this->crud->addColumn([
            'name' => 'sales_id',
            'label' => 'Assign to',
            'entity' => 'sales',
            'attribute' => 'name',
            'model' => 'App\Models\User',
            'searchLogic' => function ($query, $column, $searchTerm) {
                // Add a new where clause to perform a case-insensitive search
                $query->orWhereHas('user', function ($q) use ($searchTerm) {
                    $q->where('name', 'ILIKE', '%' . $searchTerm . '%');
                });
            },
        ]);
```

i have a column named 'sales_id.' When I search for some specific data, However, when I perform the same search in a Laravel Backpack table, it only displays 10 results,  when I use a SQL query, it shows all the matching data with total is 17 data

i use laravel backpack 4.1

i have check it generated query like this
```
SELECT *
FROM public.leads
WHERE EXISTS (
    SELECT 1
    FROM public.users
    WHERE "leads"."user_id" = "users"."id"
    AND "users"."name" ILIKE '%CYMON MAH%'
)
AND "leads"."deleted_at" IS NULL;
```
i run also in posgre it limit 10 data
but when i query like this not limit 
```

SELECT *
FROM public.leads
WHERE sales_id = 41
AND "leads"."deleted_at" IS NULL;



```

**Comment by jcastroa87** [comment]
Hello @rickyviz , sadly we didnt provided any support for Backpack v4, either way, can you say how many rows return

```
SELECT *
FROM public.users
WHERE "users"."name" ILIKE '%CYMON MAH%'
```

Maybe there you get a limit of 10 results according to match "ILIKE".

Cheers

**Comment by jcastroa87** [comment]
Hello @rickyviz, please let us know if you find any solution or if you can provide additional information to find a solution together.

Cheers

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to reopen or create a new one if needed.

Cheers.
